app/pages/lab/workflows-table.spec.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import WorkflowsTable from './workflows-table';
+
+const meta = function () {
+  return { meta: { page: 1, page_size: 1 }};
+};
+
+const workflows = [
+  { active: false, id: '1', display_name: 'Rad Workflow', getMeta: meta },
+  { active: true, id: '2', display_name: 'Cool Workflow', getMeta: meta }
+];
+
+const project = {
+  configuration: {}
+};
+
+describe('WorkflowsTable', function () {
+  let wrapper;
+  const statusChangeSpy = sinon.spy();
+  const statsVisibilityChangeSpy = sinon.spy();
+  const completenessTypeChangeSpy = sinon.spy();
+
+  before(function () {
+    wrapper = shallow(
+      <WorkflowsTable
+        labPath={(url) => { return url; }}
+        project={project}
+        handleSetStatsCompletenessType={completenessTypeChangeSpy}
+        handleWorkflowStatsVisibility={statsVisibilityChangeSpy}
+        handleWorkflowStatusChange={statusChangeSpy}
+        meta={meta}
+        workflows={workflows}
+      />
+    );
+  });
+
+  it('should display the correct amount of workflows', function () {
+    expect(wrapper.find('Link')).to.have.lengthOf(2);
+  });
+
+  it('should call handleWorkflowStatusChange on status checkbox change', function () {
+    const statusCheckbox = wrapper.find('input[id="active"]').at(0);
+    statusCheckbox.simulate('change');
+    expect(statusChangeSpy.calledOnce).to.be.true;
+  });
+
+  it('should call handleSetStatsCompletenessType on completeness-type-classification change', function () {
+    const completenessRadioButton = wrapper.find('input[value="classification"]').at(0);
+    completenessRadioButton.simulate('change');
+    expect(completenessTypeChangeSpy.calledOnce).to.be.true;
+    completenessTypeChangeSpy.reset();
```

Comment:
I'm seeing a warning about this being deprecated in the next sinon version. Could this be done in another way?